### ISSUE TITLE
PluginCompiler - Access to Sentry

### ISSUE THREAD

**Issue by nikita-tukkel** [Erdroy/Stationeers.Addons#22]
Please allow us to play with Sentry ([sentry.io](https://github.com/Erdroy/Stationeers.Addons/issues/sentry.io)), which is already embedded into the game. I think it is better to find ways of using Sentry instead of writing our own instrumentation & performance monitor.

At this moment, if I write `using Sentry;`, it will not compile.

**Comment by Erdroy** [comment]
I don't think that giving access to whole sentry is a good idea. It allows to manipulate the memory directly, adds web requests etc.
Also it exposes some sensible data (GPU id etc.), so it is better, to keep it excluded.

**Comment by nikita-tukkel** [comment]
But it is already here... Actually all I ask is the ability to write this:
```
            var itx = Sentry.SentrySdk.StartTransaction("tx", "op");
            itx.Finish();
```

Instead of this:
```
            var type = HarmonyLib.AccessTools.TypeByName("Sentry.SentrySdk");
            object itx = new HarmonyLib.Traverse(type).Method("StartTransaction", "tx", "op").GetValue();
            HarmonyLib.Traverse.Create(itx).Method("Finish").GetValue();
```

Hope you ain't going to blacklist Harmony now, are you? :-))))))))
